.github/workflows/changesets.yml

```diff
@@ -10,4 +10,15 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      # This step generates the generated/components.json file expected by the
+      # validator action
+      - name: Build
+        run: npm run build
```

Comment:
Do we need the whole build or can we run the subset `script/components-json/build`?

Reply:
@siddharthkp I think we could get away with running a minimal build, let me know if that's preferred and I can update the command here!

Reply:
I think so, but totally not a blocker, can follow up in a new PR!